examples/http-tinygo-outbound-http/readme.md

```diff
@@ -42,7 +42,7 @@ source = "main.wasm"
 allowed_http_hosts = [ "https://some-random-api.ml", "https://postman-echo.com" ]
 [component.trigger]
 route = "/hello"
-executor = { type = "wagi" }
+executor = { type = "spin" }
```

Comment:
Should we drop this since Spin is the default?

Reply:
Dropped.